src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/get_paginated_indices.js

```diff
@@ -0,0 +1,3 @@
+export function getPaginatedIndices(indices, page, perPage) {
```

Comment:
Tests for this would be great in a later iteration.

Reply:
I would maybe use the EuiPager component.  It already does the math for this for you.

Reply:
Where can I find that?

Reply:
https://github.com/elastic/eui/search?utf8=%E2%9C%93&q=EuiPager&type=

Reply:
Sorry this one's a little hidden... it's with the services: https://github.com/elastic/eui/tree/master/src/services/paging